v03_pipeline/lib/tasks/base/base_update.py

```diff
@@ -1,7 +1,7 @@
 import hail as hl
 
 from v03_pipeline.lib.misc.io import write
-from v03_pipeline.lib.tasks.base.base_hail_table_task import BaseHailTableTask
+from v03_pipeline.lib.tasks.base.base_hail_table import BaseHailTableTask
```

Comment:
I ended up doing a some renaming of the `base` tasks since we now have a `base` task that isn't an `update`.  It sprawled... happy to move the renames to a different PR so this one becomes smaller.